docs/storage-backend/cassandra.md

```diff
@@ -194,37 +194,62 @@ iterate over all vertices or edges in large graphs effectively.
-- `cert`
-- `cert.pfx`
-- `config.json`
-- `cqlshrc`
-- `identity.jks`
-- `key`
-- `trustStore.jks`
-
-The `config.json` file contains the following:
-
-- `host` - the contact point for the CQL driver
-- `port` - the CQL client port
-- `keyspace` - the name of the keyspace you created
-- `trustStoreLocation` - trustStore.jks is the truststore
-- `trustStorePassword` - your truststore password
-- `keyStoreLocation` - identity.jks is the keystore
-- `keyStorePassword` - your keystore password
-
-Using the information above, you will need to configure the following:
-
-- `storage.hostname`
-- `storage.port`
-- `storage.username`
-- `storage.password`
-- `storage.cql.keyspace`
-- `storage.cql.ssl.*`
+[Download](https://docs.datastax.com/en/astra/docs/obtaining-database-credentials.html) the secure-connect zipped bundle for your Astra database.
+
+While connecting to Astra DB from JanusGraph, it is preferred to make use of the secure bundle connection file
+as-is without extracting it. There are multiple ways in which a secure bundle connection file can be passed on to
+the JanusGraph configuration to connect to Astra DB using the DataStax driver.
+
+### Internal string configuration
+Set the following property `storage.cql.internal.string-configuration` to `datastax-java-driver { basic.cloud.secure-connect-bundle=<path-to-secure-bundle-zip-file> }`
+and set the username, password and keyspace details
+
+For eg.
+```properties
+gremlin.graph=org.janusgraph.core.JanusGraphFactory
+storage.backend=cql
+storage.cql.keyspace=<keyspace name which was created in AstraDB>
+storage.username=<clientID>
+storage.password=<clientSecret>
+storage.cql.internal.string-configuration=datastax-java-driver { basic.cloud.secure-connect-bundle=<path-to-secure-bundle-zip-file> 
+```
+
+Also, you can set a jvm argument to pass the secure bundle file as shown below and remove that property
+`(storage.cql.internal.string-configuration)` from the list above.
+
+`-Ddatastax-java-driver.basic.cloud.secure-connect-bundle=<path-to-secure-bundle-zip-file>`
+
+### Internal file configuration
+Set the following property `storage.cql.internal.file-configuration` to an external configuration file if you would like to
+externalize the astra connection related properties to a separate file and specify the secure bundle and credentials information on that file.
+
+For eg.
+```properties
+gremlin.graph=org.janusgraph.core.JanusGraphFactory
+storage.backend=cql
+storage.cql.keyspace=<keyspace-name>
+# Link to the external file that DataStax driver understands
+storage.cql.internal.file-configuration=<path-to-astra.conf>
+```
+astra.conf (external file)
+```
+datastax-java-driver {
+  basic.cloud {
+    secure-connect-bundle = "<path-to-secure-bundle-zip-file>"
+  }
+  advanced.auth-provider {
+    class = PlainTextAuthProvider
+    username = "<clientID>"
+    password = "<clientSecret>"
+  }
+}
+```
+
+_Client id and Client secret need to be generated and copied from your Astra Account
```

Comment:
If this is something you'd like to highlight, maybe prepend it with "Note:" or "Important:" instead of using italics?